eclass/rocm.eclass

```diff
@@ -176,9 +176,12 @@ _rocm_set_globals() {
 	local optflags=${allflags[@]/%/(-)?}
 	ROCM_USEDEP=${optflags// /,}
 }
-_rocm_set_globals
-unset -f _rocm_set_globals
 
+# dev-util/hip uses checkgpu but does not need the USE flags.
```

Comment:
Actually there's `dev-libs/rocm-opencl-runtime` which also need just `check-amdgpu` but not other global variables. Its approach is simply copy the `check-amdgpu` to the ebuild. Here you provide an alternative way, but restrict to `dev-util/hip`. I think it would be better to let each package decide if they want to inherit `rocm.eclass` without global variables.

Reply:
Good point.  Thanks!